test/extended/gssapi.sh

```diff
@@ -0,0 +1,290 @@
+#!/bin/bash
+#
+# TODO
+
+# set -x
+
+set -e
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+OS_ROOT=$(dirname "${BASH_SOURCE}")/../..
+source "${OS_ROOT}/hack/lib/init.sh"
+os::log::stacktrace::install
+trap os::test::junit::reconcile_output EXIT
+os::test::junit::declare_suite_start "test-extended/gssapiproxy"
+os::util::environment::setup_time_vars
+
+cd "${OS_ROOT}"
+
+os::build::setup_env
+
+hack/build-go.sh cmd/oc -tags=gssapi
+os::cmd::expect_success_and_text 'oc version' 'GSSAPI Kerberos SPNEGO'
+export OC_CMD=`which oc`
+
+function cleanup()
+{
+	out=$?
+	cleanup_openshift
+	echo "[INFO] Exiting"
+	return $out
+}
+
+trap "exit" INT TERM
+trap "cleanup" EXIT
+
+echo "[INFO] Starting server"
+
+ensure_iptables_or_die
+os::util::environment::setup_all_server_vars "test-extended/gssapiproxy"
+os::util::environment::use_sudo
+reset_tmp_dir
+
+os::log::start_system_logger
+
+configure_os_server
+
+patch='
+{
+  "oauthConfig": {
+    "identityProviders": [{
+      "name": "header",
+      "challenge": true,
+      "mappingMethod": "add",
+      "provider": {
+        "apiVersion": "v1",
+        "kind": "RequestHeaderIdentityProvider",
+        "challengeURL": "http://gssapiproxy-server.gssapiproxy.svc.cluster.local/mod_auth/oauth/authorize?${query}",
+        "headers": ["Remote-User"]
+      }
+    }]
+  }
+}
+'
+cp ${SERVER_CONFIG_DIR}/master/master-config.yaml ${SERVER_CONFIG_DIR}/master/master-config.tmp.yaml
+openshift ex config patch ${SERVER_CONFIG_DIR}/master/master-config.tmp.yaml --patch="${patch}" > ${SERVER_CONFIG_DIR}/master/master-config.yaml
+export USE_LATEST_IMAGES=true
+start_os_server
+
+export KUBECONFIG="${ADMIN_KUBECONFIG}"
+
+install_registry
+wait_for_registry
+
+oc login -u system:admin
+oc new-project gssapiproxy
+oadm policy add-scc-to-user anyuid -z default -n gssapiproxy
+
+# create all the resources we need
+oc create -f test/extended/testdata/gssapi/proxy
+
+oc create -f test/extended/testdata/gssapi/fedora/base
```

Comment:
Fair enough... when they were still `fixtures` and we did that for all the folders, maybe not so much :)